### PR TITLE
Support for custom out-of-ammo messages

### PR DESCRIPTION
**Pull request by phantamanta44**
Yeah, why was this not done before?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.*~
```
Comment (dmulloy2) [inline .gitignore]: I can't say I see the logic behind this. Wouldn't it ignore all new files?

File: src/main/java/net/dmulloy2/swornguns/types/Gun.java [hunk 2]
```diff
@@ -185,7 +186,11 @@ public final void shoot()
 			else
 			{
 				player.playSound(owner.getPlayer().getLocation(), Sound.ITEM_BREAK, 20.0F, 20.0F);
-				player.sendMessage(plugin.getPrefix() + FormatUtil.format("&eThis gun needs &b{0}&e!", ammo.getName()));
+				if (this.outOfAmmoMessage != null)
+				    player.sendMessage(plugin.getPrefix() + this.outOfAmmoMessage);
+				else {
```
Comment (dmulloy2) [inline src/main/java/net/dmulloy2/swornguns/types/Gun.java]: Formatting's a bit off here, you might as well not use brackets here.

File: .gitignore (hunk 1 above)
Comment (phantamanta44) [inline .gitignore]: It ignores files that end in a tilde, which are temporary files used by *nix operating systems.
(foobar.java~)

File: src/main/java/net/dmulloy2/swornguns/types/Gun.java (hunk 2 above)
Comment (phantamanta44) [inline src/main/java/net/dmulloy2/swornguns/types/Gun.java]: Meh. I suppose I'm just too accustomed to overusing brackets in my code.

**Pull request closed without merging**